src/runner/PlutoRunner.jl

```diff
@@ -1096,6 +1096,14 @@ function pretty_stackcall(frame::Base.StackFrame, linfo::Core.MethodInstance)
     end
 end
 
+function pretty_stackcall(frame::Base.StackFrame, linfo::Method)
+    sprint(Base.show_tuple_as_call, linfo.name, linfo.specTypes)
```

Comment:
```suggestion
    sprint(Base.show_tuple_as_call, linfo.name, linfo.sig)
```

Reply:
cool! Should we do the same for the other block a few lines above?

Do we expect `Core.MethodInstance`.`specTypes` to equal `Core.MethodInstance`.`def`.`sig`?

<img width="347" alt="image" src="https://github.com/fonsp/Pluto.jl/assets/8681967/0c936756-0478-4ae2-9c15-7c55bf5ba5f8">

Reply:
The `MethodInstance.specTypes` is more specialized than `Method.sig`, one `Method` can have multiple `MethodInstance`. For example:

```julia
julia> method = first(methods(+))
+(x::T, y::T) where T<:Union{Int128, Int16, Int32, Int64, Int8, UInt128, UInt16, UInt32, UInt64, UInt8}
     @ Base int.jl:87

julia> method_instance = first(method.specializations)
MethodInstance for +(::Int64, ::Int64)

julia> Base.show_tuple_as_call(stdout, method.name, method.sig) # abstract signature
+(::T, ::T) where T<:Union{Int128, Int16, Int32, Int64, Int8, UInt128, UInt16, UInt32, UInt64, UInt8}

julia> Base.show_tuple_as_call(stdout, method.name, method_instance.specTypes) # one specialization (::Int64, ::Int64)
+(::Int64, ::Int64)
```
so when we have the MethodInstance using specTypes gives more information.